storeys/js/core/urlresolver.js

```diff
@@ -1,6 +1,6 @@
 define(
-    ['require', 'module', '../conf/urls'],
-    function(require, module, urls) {
+    ['require', 'jquery', 'settings', 'module', 'storeys/conf/urls'],
```

Comment:
The framework part of `storeys` should not require on `jquery`. Please remove reference to jquery.

(the application can depends on jquery)